WebService.Test/v1/Models/SimulationApiModel/SimulationApiModelTest.cs

```diff
@@ -143,6 +143,28 @@ Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApi
                 .Wait(Constants.TEST_TIMEOUT);
         }
 
+        [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
+        public void ItSetsSimulationStatisticsFromServiceModel()
+        {
+            // Arrange
+            var simulation = this.GetSimulationModel();
+            var statistics = new SimulationStatisticsModel { TotalMessagesSent = 100, FailedDeviceConnectionsCount = 1, FailedDeviceTwinUpdatesCount = 2, FailedMessagesCount = 3  };
+
+            // Act
+            var result = Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel.SimulationApiModel.FromServiceModel(
```

Comment:
It would be cleaner if you could have a using so that we don't need the entire namespace here.